jersey-servers/src/main/java/com/palantir/remoting2/servers/jersey/JsonExceptionMapper.java

```diff
@@ -69,15 +61,11 @@ public final Response toResponse(T exception) {
 
         ResponseBuilder builder = Response.status(status);
         try {
-            final SerializableError error;
-            if (includeStackTrace) {
-                StackTraceElement[] stackTrace = exception.getStackTrace();
-                error = SerializableError.of(exceptionMessage, exception.getClass(),
-                        Arrays.asList(stackTrace));
-            } else {
-                error = SerializableError.of("Refer to the server logs with this errorId: "
-                        + errorId, exception.getClass());
-            }
+            SerializableError error = SerializableError.builder()
+                    .errorCode(exception.getClass().getName())
+                    .errorName("Refer to the server logs with this errorId: " + errorId)
```

Comment:
this usage of the API is different than usage by ServiceException. I think for consistency we should set errorName to `exception.getClass().getName()`.